Demo/Common/IntegrationExamples/AdvancedFlowExample.swift

```diff
@@ -0,0 +1,374 @@
+////
+//// Copyright (c) 2023 Adyen N.V.
+////
+//// This file is open source and available under the MIT license. See the LICENSE file for more info.
+////
+//
+
+import AdyenActions
+import AdyenComponents
+import AdyenDropIn
+
+internal final class AdvancedFlowExample: AdvancedFlowExampleProtocol {
+
+    internal var advancedFlowComponent: PresentableComponent?
+    internal var paymentMethods: PaymentMethods?
+    internal weak var presenter: PresenterExampleProtocol?
+
+    internal lazy var adyenActionComponent: AdyenActionComponent = {
+        let handler = AdyenActionComponent(context: context)
+        handler.configuration.threeDS.delegateAuthentication = ConfigurationConstants.delegatedAuthenticationConfigurations
+        handler.configuration.threeDS.requestorAppURL = URL(string: ConfigurationConstants.returnUrl)
+        handler.delegate = self
+        handler.presentationDelegate = self
+        return handler
+    }()
+
+    // MARK: - Initializers
+
+    internal init() {}
+
+    // MARK: - Networking
+
+    internal func requestInitialData() {
+        requestPaymentMethods(order: nil) { [weak self] paymentMethods, errorResponse in
+            guard paymentMethods != nil else {
+                guard let errorResponse = errorResponse else {
+                    return
+                }
+                self?.presentAlert(with: errorResponse) { [weak self] in
+                    self?.requestPaymentMethods(order: nil, completion: nil)
```

Comment:
Without `completion` this retry will just do nothing 🤔

Reply:
Made the change: da858bb9a858774fa139005a65bb215f94ed0c10